layouts/_default/baseof.html

```diff
@@ -3,11 +3,11 @@
 <head>
   {{ block "head" . }}{{ partial "site-head.html" .}}{{ end }}
 </head>
-<body class="bg-hexagon bg-center bg-contain">
+<body>
 
   {{ block "header" . }}{{ partial "site-header.html" .}}{{ end }}
 
-  <main>
+  <main class="pb-36">
```

Comment:
This was added to provide space between the end of the main content and the footer. I'm sure this will likely be updated as changes continue to be made.